doc/versions.json

```
@@ -27,19 +27,19 @@
   ],
   "COMPONENTS_BY_VERSION": {
     "latest": {
-      "ncs": "1.9.99",
-      "nrfxlib": "1.9.99",
-      "zephyr": "2.7.99",
-      "mcuboot": "1.8.99",
+      "ncs": "2.0.0",
+      "nrfxlib": "2.0.0",
+      "zephyr": "3.0.99",
+      "mcuboot": "1.9.99",
       "nrfx": "2.7",
       "tfm": "1.5.0",
       "matter": ""
     },
-    "1.9.99-dev1": {
-      "ncs": "1.9.99-dev1",
-      "nrfxlib": "1.9.99",
-      "zephyr": "2.7.99",
-      "mcuboot": "1.8.99",
+	"2.0.0": {
```

I think there is a problem with the tab space.
```suggestion
    "2.0.0": {
      "ncs": "2.0.0",
```